### PR TITLE
[master < T208-MAGE] Add Louvain community detection documentation

### PR DESCRIPTION
**Pull request by antepusic**
Documentation for MAGE’s Louvain community detection query module (https://github.com/memgraph/mage/pull/48).

- [x] Query module documentation
- [x] Algorithm description

### CODE REVIEW

File: mage/query-modules/cpp/community_detection.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+    {children}
+  </span>
+);
+
+[![docs-source](https://img.shields.io/badge/source-community_detection_online-FB6E00?logo=github&style=for-the-badge)](https://github.com/memgraph/mage/blob/main/cpp/community_detection_module/community_detection_module.cpp)
+
+## Abstract
+
+This query module makes available the [Louvain method](https://arxiv.org/abs/1305.2006)
+<sup>[1]</sup> for community detection, using the [Grappolo](https://github.com/Exa-Graph/grappolo)
```
Comment (jmatak) [inline mage/query-modules/cpp/community_detection.md:27]:
```suggestion
This query module makes available the [Louvain method](https://arxiv.org/abs/1305.2006) <sup>[1]</sup> for community detection, using the [Grappolo](https://github.com/Exa-Graph/grappolo)
```

Comment (jmatak) [reply mage/query-modules/cpp/community_detection.md:27]: The formatting was weird before removing this newline

**Review by jmatak — CHANGES_REQUESTED** [review]
One small little detail that needs to be fixed, otherwise :rocket:

File: mage/query-modules/cpp/community_detection.md (hunk 1 above)
Comment (antepusic) [reply mage/query-modules/cpp/community_detection.md]: Thanks!

**Review by jmatak — APPROVED** [review]

**Review by antepusic — COMMENTED** [review]
Why not italicize complexity  (relevant [commit](https://github.com/memgraph/docs/pull/350/commits/c88849c76e0cae3354861f1361585c8df015e1ba)) in query module READMEs? Using a different text style makes them more visible to the reader.

**Review by g-despot — APPROVED** [review]

**Pull request merged**